target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1 @@
+/Users/ambichaliki/Documents/automation/src/main/java/com/ff/psa/App.java
```

Comment:
Please make sure that the target directory is ignored and do not check into the repository.
This applies to all the target folders.

Reply:
Done